packages/wdio-browserstack-service/src/ai-handler.ts

```diff
@@ -0,0 +1,212 @@
+import aiSDK from '@browserstack/ai-sdk-node'
+import { BStackLogger } from './bstackLogger.js'
+import { TCG_URL, TCG_INFO, SUPPORTED_BROWSERS_FOR_AI } from './constants.js'
+import { handleHealingInstrumentation } from './instrumentation/funnelInstrumentation.js'
+import { createRequire } from 'node:module'
+
+import type { Capabilities } from '@wdio/types'
+import type BrowserStackConfig from './config.js'
+import type { Options } from '@wdio/types'
+import type { BrowserstackHealing } from '@browserstack/ai-sdk-node'
+import path from 'node:path'
+import fs from 'node:fs'
+import url from 'node:url'
+import { getBrowserStackUserAndKey, isBrowserstackInfra } from './util.js'
+import type { BrowserstackOptions } from './types.js'
+
+class AiHandler {
+    authResult: BrowserstackHealing.InitSuccessResponse | BrowserstackHealing.InitErrorResponse
+    wdioBstackVersion: string
+    constructor() {
+        this.authResult = JSON.parse(process.env.TCG_AUTH_RESULT || '{}')
```

Comment:
can we avoid passing json using env vars ?

Reply:
We need to pass the results from `aiSDK.BrowserstackHealing.init` call made from launcher to the service. So, in case we want to avoid passing json via env vars, we will have to pass multiple env vars which is less efficient than `JSON.stringify` and `JSON.parse`